docs/extensibility/extend-rest-api-update-cart.md

```diff
@@ -0,0 +1,90 @@
+# Updating the cart with the Store API
+If you're an extension developer, and your extension does some server-side processing as a result of some client-side
+input, then you may want to use the `cart/extensions` endpoint to signal to
+the Store API that some processing needs to take place.
+
+Extensions may register a callback to run when the `cart/extensions` endpoint is hit. When it is, the server will
+execute all registered callbacks for a specified namespace.
+
+## Example use case
+You are the author of an extension that automatically applies valid coupons to a user's cart when they press a
+button.
+
+Your extension adds a button to the sidebar in the Cart and Checkout blocks using the [`DiscountsMeta`](./available-slot-fills.md) Slot.
+
+When this button is clicked, you want the server to do some work to figure out which coupons the customer is eligible
+for, and then apply them to the cart.
+
+When this is done, you expect to see the updated cart data in the client.
```

Comment:
This fits very well in the intro of this document and severs as a good introduction to what problem are you solving here. After an initial read, developers can just skip that section straight into basic usage or API definition.

Reply:
Fixed by the new format, I think.